mosqagent.c

```diff
@@ -105,6 +107,100 @@ void mqtta_dispose_message(struct mqtta_message *msg)
     free(msg);
 }
 
+/**
+ * If there is already a configuration in the agent, make sure to take care
+ * of the memory object before calling this function, otherwise there may be
+ * memory leaks!
+ */
+int mqtta_load_configuration(struct mosqagent *agent,
```

Comment:
The same Doxygen comment in both declaration and implementation tends to drift apart over time.